docs/find_color_card.md

```diff
@@ -33,7 +33,7 @@ rgb_img, path, filename = pcv.readimage("target_img.png")
 df, start, space = pcv.transform.find_color_card(rgb_img=rgb_img)
 
 # Use these outputs to create a labeled color card mask
-mask = pcv.transform.create_color_card_mask(rgb_img=img, radius=10, start_coord=start, spacing=space, ncols=6, nrows=4, label="prefix")
+mask = pcv.transform.find_color_card(rgb_img=img, radius=10, start_coord=start, spacing=space, ncols=6, nrows=4, label="prefix")
```

Comment:
The description in the issue I wrote wasn't clear, I meant that the function was correct (`create_color_card_mask`), but the `label="prefix"` input should move up to the `find_color_card` step